packages/palette/src/elements/Image/LazyImage.tsx

```diff
@@ -64,14 +65,63 @@ export const LazyImage: React.FC<LazyImageProps> = ({
     borderRadius,
     style,
     onError,
+    blurhash,
     ...containerProps
   } = props
 
   if (preload) {
-    return <ImageComponent {...props} />
+    return (
+      <SkeletonImage
+        width={width}
+        height={height}
+        borderRadius={borderRadius}
+        blurhash={hideBlurhash ? undefined : blurhash}
+        {...containerProps}
+      >
+        <ImageComponent onLoad={() => setHideBlurhash(true)} {...props} />
+      </SkeletonImage>
+    )
```

Comment:
Not sure if we should do it here as well, but did it just in case, but it looks different than the lazy loaded one


https://github.com/artsy/palette/assets/20655703/91dda4d6-82fb-49d7-bcef-f0a306e4fa3b

Reply:
The difference just being that the full-res images "wipes" in over the blurhash? Seems fine to me.